test/cri-containerd/container_test.go

```diff
@@ -305,6 +305,6 @@ func Test_RunContainer_VirtualDevice_GPU_LCOW(t *testing.T) {
 		t.Fatalf("expected to see no error, instead saw %s", string(response.Stderr))
 	}
 	if len(response.Stdout) == 0 {
-		t.Fatal("expected to see GPU device on container, not present")
+		t.Fatalf("expected to see GPU device on container, not present")
```

Comment:
This should just be `Fatal`, right?

Reply:
Technically yes, weird that it compiles like this haha

Reply:
Done.